github-ops.yml

```diff
@@ -19,8 +19,9 @@ jobs:
     steps:
       - name: Ping runner workflow
         env:
-          # only needs public repo scope (classic token)
+          # only needs github actions (write) `workflow_dispatch`
           GH_OPS_PAT: ${{ secrets.GH_OPS_PAT }}
```

Comment:
Note that we currently have one repo PAT named `GH_PAT` configured. `secrets.GH_OPS_PAT` does not exist in this repo.

```
Environment secrets: There are no secrets for this repository’s environments.
Repository secrets: GH_PAT Updated 2 hours ago
Organization secrets
```

Reply:
Yep. This one is meant for repos like zkevm-chain,zkevm-circuits to trigger the workflows faster. It's optional tho